scripts/image-builder/config/kickstart.ks.template

```diff
@@ -1,13 +1,33 @@
 lang en_US.UTF-8
 keyboard us
 timezone UTC
-zerombr
-clearpart --all --initlabel
-autopart --type=plain --fstype=xfs --nohome
-reboot
 text
+reboot
+
+# Configure network to use DHCP and activate on boot
 network --bootproto=dhcp --device=link --activate --onboot=on
 
+# Partition disk with a 1GB boot XFS partition and an LVM volume containing a 5GB+ system root
+# The remainder of the volume will be used by the CSI driver for storing data
+#
+# For example, a 20GB disk would be partitioned in the following way:
+#
+# NAME          MAJ:MIN RM SIZE RO TYPE MOUNTPOINT
+# sda             8:0    0  20G  0 disk 
+# ├─sda1          8:1    0   1G  0 part /boot
+# └─sda2          8:2    0  19G  0 part 
+#  └─rhel-root  253:0    0   5G  0 lvm  /sysroot
+#
+zerombr
+clearpart --all --initlabel
+part /boot --fstype=xfs --asprimary --size=1024
+# Uncomment this line to add a SWAP partition of the recommended size
+#part swap --fstype=swap --recommended
+part pv.01 --grow
+volgroup rhel pv.01
+logvol / --vgname=rhel --fstype=xfs --size=REPLACE_LVM_SYSROOT_SIZE --name=root
+
+# Configure ostree
 ostreesetup --nogpg --osname=rhel --remote=edge --url=file:///run/install/repo/ostree/repo --ref=rhel/8/x86_64/edge
```

Comment:
```suggestion
ostreesetup --nogpg --osname=rhel --remote=edge --url=file:///run/install/repo/ostree/repo --ref=rhel/8/x86_64/edge
```

Unrelated to this patch, just a comment, I wonder if there is a way to dynamically use the right arch (uname -m) on a quickstart: x86_64 vs aarch64